### PR TITLE
chore(docs): Fixed rendering of a few code snippets

### PR DESCRIPTION
**Pull request by alex**
reST requires 2 backticks, not 3

### CODE REVIEW

**Comment by deis-admin** [comment]
Thanks for the contribution! Please ensure your commits follow our [style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). This code will be tested once a Deis maintainer reviews it.

**Comment by sgoings** [comment]
Thanks for fixing up the docs for us! I'd suggest rewording your commit message to:

`chore(docs): fix rendering of a few code snippets`

**Comment by alex** [comment]
@sgoings done! thanks.

File: docs/managing_deis/security_considerations.rst [hunk 1]
```diff
@@ -40,13 +40,13 @@ Users of Deis often want to deploy their applications to separate environments
...
+In Deis, deployed applications can be segregated by using the ``deis tags`` command. This
 enables you to tag machines in your cluster with arbitrary metadata, then configure your applications
 to be scheduled to machines which match the metadata.
 
-For example, if some machines in your cluster are tagged with ```environment=production``` and some
-with ```environment=staging```, you can configure an application to be deployed to the production
-environment by using ```deis tags set environment=production```. Deis will pass this configuration
+For example, if some machines in your cluster are tagged with ``environment=production`` and some
+with ``environment=staging``, you can configure an application to be deployed to the production
+environment by using ``deis tags set environment=productio``. Deis will pass this configuration
```
Comment (bacongobbler) [inline docs/managing_deis/security_considerations.rst]: looks like a typoi made it in here :)

**Comment by carmstrong** [comment]
ping @alex - are you able to fix the typo? We can get this merged. Thanks!

**Comment by alex** [comment]
@carmstrong Fixed, sorry about that, somehow I missed the notification.

**Comment by mboersma** [comment]
LGTM.

**Comment by alex** [comment]
@mboersma Done! Sorry about that.

**Comment by mboersma** [comment]
Thanks @alex! LGTM.

**Comment by bacongobbler** [comment]
lgtm!
